test/basic-querying.test.js

```diff
@@ -172,7 +186,9 @@ describe('basic-querying', function() {
       User.find({skip: 1, limit: 4, order: 'seq'}, function(err, users) {
         should.exists(users);
         should.not.exists(err);
-        users[0].seq.should.be.eql(1);
+        bdd.itIf(!isCassandra, 'in-place sort is supported', function() {
+          users[0].seq.should.be.eql(1);
+        });
```

Comment:
ah, overused the bdd.itIf convention.  reverted all of them except for describe.